integration_tests/emukit/benchmarking/test_benchmarker.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import emukit.test_functions
+from emukit.benchmarking.benchmarker import Benchmarker
+from emukit.benchmarking.metrics import MinimumObservedValueMetric
+from emukit.core import ContinuousParameter
+from emukit.examples.gp_bayesian_optimization.single_objective_bayesian_optimization import GPBayesianOptimization
```

Comment:
Given that we will probably hold a lower bar for examples folder, let's not rely on it for unit testing core functionality of the library